forum/qa-theme/SnowFlat/js/selectLanguage.js

```diff
@@ -0,0 +1,111 @@
+if (typeof CKEDITOR != 'undefined') {
+    CKEDITOR.on( "instanceReady", function() {
+        const codeBlock = document.getElementById( "cke_43" );
+        if ( location.href.includes( "/ask" ) ) {
+            codeBlock.addEventListener( "click", function() {
+                const category_1 = $('#category_1 option:selected');

```

Comment:
Jeśli element pobierany do zmiennej **category_1** nie jest dodawany/usuwany do DOM za każdym razem, gdy dodaje się bloczek z kodem, a po prostu pokazywany/ukrywany (lub raz tworzony, a potem ukrywany), to można go pobierać wcześniej i raz, a nie na każde kliknięcie - oszczędzamy liczbę odniesień do DOM.